jpo-asn-runtime/src/main/java/us/dot/its/jpo/asn/runtime/serialization/EnumeratedSerializer.java

```diff
@@ -0,0 +1,52 @@
+package us.dot.its.jpo.asn.runtime.serialization;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import com.fasterxml.jackson.dataformat.xml.ser.XmlSerializerProvider;
+import java.io.IOException;
+import us.dot.its.jpo.asn.runtime.types.Asn1Enumerated;
+
+/**
+ * Base class for ENUMERATED value serializers to produce both XER and JER
+ * @param <T> The ENUMERATED type
+ * TODO: this doesn't need to be generic
+ * @author Ivan Yourshaw
+ */
+@SuppressWarnings({"unchecked", "rawtypes"})
+public class EnumeratedSerializer<T extends Enum & Asn1Enumerated> extends StdSerializer<T> {

```

Comment:
suggestion: use typing from the EnumeratedDeserializer to avoid suppressing warnings
```suggestion
/**
 * Base class for ENUMERATED value serializers to produce both XER and JER
 * @param <T> The ENUMERATED type
 * @author Ivan Yourshaw
 */
public class EnumeratedSerializer<T extends Enum<?> & Asn1Enumerated> extends StdSerializer<T> {
```

Reply:
That works. It is fixed, thanks.